pom.xml

```diff
@@ -93,6 +93,11 @@
       <artifactId>jsr305</artifactId>
       <version>3.0.1</version>
     </dependency>
+		<dependency>
+		    <groupId>org.json</groupId>
```

Comment:
I sort of half read the other json related thread you have going. Are we switching this to gson?